src/main/java/core/basesyntax/figure/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.figure;
+
+import core.basesyntax.supplier.Color;
+
+public class IsoscelesTrapezoid extends Figure {
+    private Color color;
```

Comment:
This field already exists in Figure class

Reply:
fixed